app/templates/index.hbs

```diff
@@ -9,17 +9,13 @@
       <div class="col--6-12">
         <h3 class="h5 u-spacer--tiny">Kies een bestuurseenheid</h3>
         <p class="small u-spacer--tiny">Zoals een gemeentebestuur, autonoom gemeentebedrijf, OCMW-vereniging, intergemeentelijk samenwerkingsverband (IGS), provinciebestuur of autonoom provinciebedrijf (APB)</p>
-        {{select-bestuurseenheid
-          value=bestuurseenheidId
-          onSelectionChange=(action "setBestuurseenheidId")
-          class="u-spacer--tiny ember-power-select--large"
-        }}
+        <SelectBestuurseenheid @value={{bestuurseenheidId}} @onSelectionChange={{action "setBestuurseenheidId"}} @class="u-spacer--tiny ember-power-select--large" />
```

Comment:
Don't use the `{{action}}` helper in templates anymore (also occurs in other templates)